hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/ApplyMCConfigMessageTask.java

```diff
@@ -35,12 +35,17 @@ public ApplyMCConfigMessageTask(ClientMessage clientMessage, Node node, Connecti
 
     @Override
     protected Object call() throws Exception {
-        ManagementCenterService mcService = nodeEngine.getManagementCenterService();
+        ManagementCenterService mcs = nodeEngine.getManagementCenterService();
+        if (mcs == null) {
+            // as ManagementCenterService is not initialized yet,
+            // we silently ignore this operation and let MC to retry later
```

Comment:
It might be a good idea to throw an exception here to let MC know that the call was not successful. IIRC there were a few places where we ignored MCService not being ready. I think we should throw in those cases as well if that's the case.

Reply:
Makes sense. I've added a throw statement here and checked if that's necessary in other ops.